lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -332,7 +332,7 @@ def create_tar(cv, cvv, repositories, json)
               repo_dirs.push(repo['relative_path'])
             end
 
-            `tar cvfh #{repo_tar} #{repo_dirs.join(" ")}`
+            `tar --selinux -cvfh #{repo_tar} #{repo_dirs.join(" ")}`
```

Comment:
this errors and needs to be `tar --selinux -cvfh #{repo_tar} #{repo_dirs.join(" ")} --selinux` I think, testing it now

Reply:
Updated